### PR TITLE
AST-579 Bug in searchform.php translation

### PR DESCRIPTION
**Pull request by murshed**
Bug in searchform.php translation

### CODE REVIEW

**Review by Nikschavan — DISMISSED** [review]

**Review by premanshup — DISMISSED** [review]

**Review by imnavanath — DISMISSED** [review]

**Review by sagarb90 — DISMISSED** [review]

**Comment by premanshup** [comment]
@murshed Thank you for the PR, this will be released in the version v3.7.0.

**Comment by murshed** [comment]
I have modified the code.
If we added 'astra" on the line the translation does not work! Video- https://streamable.com/sj2utg

File: searchform.php [hunk 1]
```diff
@@ -21,6 +21,6 @@
 			</button>
 		<?php } ?>
 	</label>
-	<input type="submit" class="search-submit" value="Search">
+	<input type="submit" class="search-submit" value="<?php echo esc_attr_x( 'Search', 'submit button' ); ?>" />
```
Comment (premanshup) [inline searchform.php:24]:
```suggestion
	<input type="submit" class="search-submit" value="<?php echo esc_attr_x( 'Search', 'submit button', 'astra' ); ?>" />
```

**Review by premanshup — CHANGES_REQUESTED** [review]

**Comment by premanshup** [comment]
> I have modified the code.
> If we added 'astra" on the line the translation does not work! Video- https://streamable.com/sj2utg

@murshed For some reason the link you shared is not working and I think it can still be translated using Loco Translate plugin. Since it will not appear the translatable strings.

**Pull request merged**